tests/components/hydrawise/conftest.py

```diff
@@ -1,6 +1,6 @@
 """Common fixtures for the Hydrawise tests."""
 
-from collections.abc import Awaitable, Callable
+from collections.abc import Awaitable, Callable, Generator
```

Comment:
This change is incorrect (probably linked to #119900).
Please keep `Generator` from `typing_extensions`.
```suggestion
from collections.abc import Awaitable, Callable
```

Reply:
Fixed.